### DISCUSSION TITLE
ImageMagick Windows11 intall problem

### DISCUSSION THREAD

**Discussion by Bizzy-Shz** [ImageMagick/ImageMagick#7425] [Help]
### ImageMagick version

7.1.1-34 Q16 x64 b0b7b17:20240623

### Operating system, version and so on

Windows 11 Pro, 23H2, 64-Bit, AMD Ryzen 5 PRO 3400GE

### Description

I installed the newest ImageMagick.exe and can't run the test cmd. I testet other cmd's and it seems that IM has no permissions and/or can't find the Modules.

![Screenshot 2024-06-25 104119](https://github.com/ImageMagick/ImageMagick/assets/110717923/8570a5e8-1b06-4d99-9473-b7d3ab5e1f3a)

![Screenshot 2024-06-25 103335](https://github.com/ImageMagick/ImageMagick/assets/110717923/9e065442-1af7-4a8c-b320-0e135aa90708)

**Comment by dlemstra** [comment]
I suspect this is something on your machine because I am unable to reproduce this issue with a clean install in a Virtual Machine that is using Windows 11 Pro.

**Comment by snibgo** [comment]
@Bizzy-Shz:

Don't run ImageMagick in `C:\Windows\System32`. You may not have permission to create files there. If you do have permission, you might destroy your operating system. Instead, create another directory and do your work there.

Please show the text output from `magick -list format`.

> I installed the newest ImageMagick.exe

How did you install it?

@Suzanaralu: Please do not repeat your question across multiple threads. That is called "spamming".

**Comment by Bizzy-Shz** [comment]
I switched over to my Macbook and there was no problem, thanks anyway
